api/utils/aws/endpoint.go

```diff
@@ -576,8 +582,7 @@ const (
 // CassandraEndpointURLForRegion returns a Cassandra endpoint based on the provided region.
 // https://docs.aws.amazon.com/keyspaces/latest/devguide/programmatic.endpoints.html
 func CassandraEndpointURLForRegion(region string) string {
```

Comment:
We might need to consider the fips endpoints here as well.
Sorry if this is out of scope

Reply:
This is a helper function that makes a URL based on the region if URI is empty. No fips endpoint is preferred but a  a user can provide fips  endpoint explicitly: `cassandra.us-east-1.amazonaws.com:9142`